ipykernel/eventloops.py

```diff
@@ -125,7 +125,16 @@ def loop_qt4(kernel):
 @register_integration('qt', 'qt5')
 def loop_qt5(kernel):
     """Start a kernel with PyQt5 event loop integration."""
-    os.environ['QT_API'] = 'pyqt5'
+    if os.environ.get('QT_API', None) is None:
+        try:
+            import PyQt5
+            os.environ['QT_API'] = 'pyqt5'
+        except ImportError:
+            try:
+                import PySide2
+                os.environ['QT_API'] = 'pyside2'
+            except ImportError:
+                pass
```

Comment:
I think we should set `QT_API='pyqt5'` at this point to have the same behavior as before?

Reply:
I'm super necromancing this PR - but are there circumstances where there's any effect to setting `QT_API='pyqt5'` even though both   `import PyQt5` and `import pyside2` have failed? If so, then we can add that in place of the path, but I'm thinking maybe this is OK as is?